### PR TITLE
Implement test listener

### PR DESCRIPTION
**Pull request by 9a24f0**
I opened PR for better tracking.

### CODE REVIEW

File: tests/test_listener.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
+# Source pytest module
```
Comment (McSinyx) [inline tests/test_listener.py]: Typo here

File: tests/test_listener.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+from math import inf
+
+
+def test_gain(context):
+    """Test write property gain."""
+    Listener(context).gain = 5/7
+    Listener(context).gain = 7/5
+    Listener(context).gain = 0
+    Listener(context).gain = inf
+    with raises(ValueError): Listener(context).gain = -1
```
Comment (McSinyx) [inline tests/test_listener.py]: Please use `context.listener` instead.

File: tests/test_listener.py [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    Listener(context).orientation = [(420, 69, 420), (69, -420, 0)]
+
+
+def test_meters_per_unit(context):
+    """Test write property meter_per_unit."""
+    Listener(context).gain = 4/9
+    Listener(context).gain = 9/4
+    Listener(context).gain = 0
+    Listener(context).gain = inf
+    with raises(ValueError): Listener(context).gain = -1
```
Comment (McSinyx) [inline tests/test_listener.py]: Mismatching function name-body.

File: tests/test_listener.py [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+
+def test_position(context):
+    """Test write property position."""
+    Listener(context).position = (1, 0, 1)
+    Listener(context).position = (1, 0, -1)
+    Listener(context).position = (1, -1, 0)
+    Listener(context).position = (1, 1, 0)
+    Listener(context).position = (0, 0, 0)
+    Listener(context).position = (1, 1, 1)
```
Comment (McSinyx) [inline tests/test_listener.py]: `context.listener.position = 1, 1, 1` would look a lot cleaner; the same with velocity.

File: tests/test_listener.py [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+
+def test_orientaion(context):
+    """Test write property orientation."""
+    Listener(context).orientation = [(420, 0, 69), (0, 42, 0)]
+    Listener(context).orientation = [(69, 0, -420), (0, -69, 420)]
+    Listener(context).orientation = [(0, 420, -69), (420, -69, 69)]
+    Listener(context).orientation = [(0, 0, 42), (-420, -420, 0)]
+    Listener(context).orientation = [(0, 0, 0), (-420, -69, -69)]
+    Listener(context).orientation = [(420, 69, 420), (69, -420, 0)]
```
Comment (McSinyx) [inline tests/test_listener.py]: `context.listener.orientation = (420, 69, 420), (69, -420, 0)` would be cleaner.  Also, nice.

**Review by McSinyx — CHANGES_REQUESTED** [review]

File: tests/test_listener.py (hunk 1 above)
Comment (9a24f0) [reply tests/test_listener.py]: Wait, we have "Source pytest module" for `test_context` also LOL

Comment (McSinyx) [reply tests/test_listener.py]: Probably missed that, fix it then.

**Review by McSinyx — APPROVED** [review]
LGTM, you can squash and merge any time.

File: tests/test_listener.py (hunk 1 above)
Comment (McSinyx) [reply tests/test_listener.py]: Could you please add more test and also fix the typo in `test_context`?

**Pull request merged**